nginx/ngx_http_js_module.c

```diff
@@ -2972,6 +2972,38 @@ ngx_http_js_request_variables(njs_vm_t *vm, njs_object_prop_t *prop,
     name.len = val.length;
 
     if (setval == NULL) {
+        #if (NGX_PCRE)
+        is_capture = 1;
+        for (size_t i = 0; i < name.len; i++) {
+            if (name.data[i] < '0' || name.data[i] > '9') {
+                is_capture = 0;
+                break;
+            }
+        }
+
+        if (is_capture) {
+            /*
```

Comment:
The commit seems excessive here, because we have similar places with no such comments. It is a basic PCRE API knowledge.